src/app/views/datasets-new/edit-metadata/DatasetMetadata.jsx

```diff
@@ -219,28 +219,15 @@ const DatasetMetadata = ({
                 legend={"National Statistic"}
                 disabled={disableForm}
             />
-
-            <RadioGroup
-                groupName="national-statistic"
-                radioData={[
-                    {
-                        id: "national-statistic-yes",
-                        value: "true",
-                        label: "Yes",
-                    },
-                    {
-                        id: "national-statistic-no",
-                        value: "false",
-                        label: "No",
-                    },
-                ]}
-                selectedValue={metadata.nationalStatistic ? metadata.nationalStatistic.toString() : "false"}
-                onChange={handleNationalStatisticChange}
-                inline={true}
-                legend={"National Statistic"}
-                disabled={disableForm}
-            />
-
+            <h2>Contact details</h2>
+            <Input
+                id="contact-name"
+                name="contactName"
+                label="Contact name"
+                onChange={handleStringInputChange}
+                value={metadata.contactName}
+                disabled={props.disableForm}
```

Comment:
I wonder if this should be `disableForm` like in other inputs?

```suggestion
                disabled={disableForm}
```

Reply:
of course